### PR TITLE
remove fv3gfs-wrapper, fv3gfs-fortran, and wrapper docker infrastructure

### PR DESCRIPTION
**Pull request by mcgibbon**
## Purpose

Previously we added fv3gfs-wrapper and fv3gfs-fortran under fv3core/external to support docker images using both packages. We have since stopped supporting this configuration. The files from these submodules are greatly increasing our file count on our HPC testing system.

This PR removes all unsupported configurations of the wrapper, along with the fv3gfs-wrapper and fv3gfs-fortran submodules.

## Infrastructure changes:

- Removed external/fv3gfs-wrapper and external/fv3gfs-fortran
- Removed all fortran and wrapper make targets
- Removed documentation of removed wrapper functionality
- Removed fv3core/examples/wrapped showing examples of wrapper-dycore runfiles

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
- [x] Docstrings and type hints are added to new and updated routines, as appropriate
- [x] All relevant documentation has been updated or added (e.g. README, CONTRIBUTING docs)
- [x] For each public change and fix in `pace-util`, HISTORY has been updated

### CODE REVIEW

**Comment by elynnwu** [comment]
launch jenkins

File: fv3core/docker/dependencies.Dockerfile [hunk 1]
```diff
@@ -158,95 +158,3 @@ RUN git config --global http.sslverify false && \
     cd NCEPlibs && \
```
Comment (elynnwu) [inline fv3core/docker/dependencies.Dockerfile:158]: Remove NCEPlibs too

File: fv3core/docker/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,8 @@
-ARG FORTRAN_IMAGE
 ARG FMS_IMAGE
 ARG ESMF_IMAGE
 ARG SERIALBOX_IMAGE
 ARG ENVIRONMENT_IMAGE
 
-FROM $FORTRAN_IMAGE as fortran_image
 FROM $ESMF_IMAGE as esmf_image
 FROM $FMS_IMAGE as fms_image
```
Comment (elynnwu) [inline fv3core/docker/Dockerfile:7]: These two lines can also be deleted

**Review by elynnwu — COMMENTED** [review]
Thanks for doing this!

File: fv3core/docker/Dockerfile (hunk 2 above)
Comment (mcgibbon) [reply fv3core/docker/Dockerfile:7]: Deleted.

File: fv3core/docker/dependencies.Dockerfile (hunk 1 above)
Comment (mcgibbon) [reply fv3core/docker/dependencies.Dockerfile:158]: I thought we might use `fv3gfs-environment`, but it doesn't look like it. Deleted that whole stage.

**Review by elynnwu — APPROVED** [review]

**Comment by elynnwu** [comment]
launch jenkins

**Comment by mcgibbon** [comment]
launch jenkins

**Pull request merged**